packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -20,6 +19,35 @@ const getSnap = (scrollSnap: ScrollSnap) => {
   return scrollSnap;
 };
 
+const getTriggerOffset = (spacing: string) => {
```

Comment:
Tbh, I'm not sure if it's the best approach (probably not (?)), however, I needed to get the pixel value of set spacing and use this value as an integer for `TRIGGER_OFFSET` const. This value is used for computations in `handleScroll` functions, which is crucial for setting the visibility of arrow buttons.